.github/workflows/docs_deploy.yml

```
@@ -0,0 +1,17 @@
+  workflow_dispatch:
+jobs:
+  deploy-docs:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v5
+        with:
+          python-version: 3.x
+      - run: pip install mkdocs-material
+      - run: pip install mkdocs-minify-plugin
+      - run: mkdocs gh-deploy --force
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)